packages/@angular/cli/package.json

```diff
@@ -9,7 +9,7 @@
   },
   "keywords": [
     "angular",
-    "angular-cli"
+    "Angular CLI"
```

Comment:
Keep the old value as well (but add the new one) so that people can search for it and find this package.